webapp/sources/rudder/rudder-rest/src/test/resources/logback-test.xml

```diff
@@ -0,0 +1,85 @@
+<configuration>
```

Comment:
This file is needed to avoid millions of logs lines during tests of rudder rest about `jgit`